dev-docs/roadmaps/gpgpu-roadmap.md

```diff
@@ -134,6 +130,10 @@ Actions
 * Fix any remaining bugs in TransformFeedback / Buffer classes
 * Update any TF code/PRs: Wind example and Attribute animation
 
+### Transform : Improved Transformfeedback API (Shipped in v5.2)
 
+This new class provides an easy-to-use interface to Transform Feedback. It hides WebGL API complexity by internally creating and managing all required WebGL objects that are necessary to perform Transform Feedback operations. New demos and existing features (AttributeTransitions) are updated to use this newer class to reduce code complexity.
 
+### ScreenGridLayer with GPU based grid aggregation. (deck.gl, shipped in v6.0)
```

Comment:
Maybe put these in reverse chronological order and add a `##` heading per release